models/fivetran_platform__user_audit.sql

```diff
@@ -0,0 +1,93 @@
+with logs as (
+
+    select 
+        *,
+        {{ fivetran_utils.json_parse(string='message_data', string_path=['actor']) }} as actor_email
+    from {{ ref('stg_fivetran_platform__log') }}
+),
+
+user_logs as (
+
+    select *
+    from logs
+    where actor_email is not null 
+        and lower(actor_email) != 'fivetran'
```

Comment:
Can you explain this and statement? Is this to filter our fivetran system events?

Reply:
yeah so there are some fivetran-triggered events where the `actor` is just `Fivetran` (many other Fivetran events simply don't have an `actor` attribute). in our own test data, these Fivetran-actor log events are all like this
<img width="617" alt="image" src="https://github.com/fivetran/dbt_fivetran_log/assets/65564846/583a236c-de44-4a13-a940-2f0a9b69ad29">


since the purpose of this model is audit user activity, i figured actions that are not actually attached to users should get the boot, but very open to opinions

Reply:
That's what I assumed. Thanks for clarifying and I agree we should filter those out as I don't imagine users would be too interested in Fivetran actor events.